sdk/python/feast/feature_server.py

```diff
@@ -246,3 +262,48 @@ def start_server(
 
         app = get_app(store, registry_ttl_sec)
         uvicorn.run(app, host=host, port=port, access_log=(not no_access_log))
+
+
+def _get_features_from_body(store: "feast.FeatureStore", body: Request):
+    body = json.loads(body)
+    # Initialize parameters for FeatureStore.get_online_features(...) call
+    if "feature_service" in body:
+        features = store.get_feature_service(body["feature_service"], allow_cache=True)
+    else:
+        features = body["features"]
+    return features, body
+
+
+def _get_predictions(
```

Comment:
@tokoko i can just move this to `store.infer`

Reply:
Yeah, sounds about right.